### PR TITLE
data : BaruReddy

### PR DESCRIPTION
**Pull request by baruKreddy**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5504"></a>

### CODE REVIEW

File: data/baruKreddy.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Bhargavi Reddy K",
+  "type": "personal",
+  "bio": "**OpenSource Developer**. Travel enthusiast & People Lover"
```
Comment (khushalarora) [inline data/baruKreddy.json:4]: Hi @baruKreddy, please add a **'Comma'** at the end of this line. :+1:

File: data/baruKreddy.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "links": [
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/bhargavikreddy/",
+      "icon": "FaLinkedin"
+    }
```
Comment (khushalarora) [inline data/baruKreddy.json:10]: Could you add a **'Comma'** here also :+1:

**Review by khushalarora — CHANGES_REQUESTED** [review]

File: data/baruKreddy.json (hunk 1 above)
Comment (baruKreddy) [reply data/baruKreddy.json]: Sure thing, Thank you :)

**Review by khushalarora — APPROVED** [review]
Thank you :tada:

**Review by bkpecho — APPROVED** [review]
Thanks for your contribution! @baruKreddy 🎉

Good luck on your open-source journey! I hope you have a great day ahead.

**Review by amandamartin-dev — APPROVED** [review]

**Pull request merged**